i18n/fr_FR.ts

```diff
@@ -5,51 +5,51 @@
     <message>
         <location filename="main_controller.py" line="70"/>
         <source>Show &amp;Filters</source>
-        <translation type="unfinished"></translation>
+        <translation>Montrer les filtres</translation>
```

Comment:
It is necessary to leave a symbol "&amp;", it indicates what character is used for quick access